en/installation.rst

```diff
@@ -138,6 +138,25 @@ Be aware that is not recommended, as your application can break when the next ma
 version is being released. Additionally composer does not cache development
 branches, so it slows down consecutive composer installs/updates.
 
+Installation using Oven
+-----------------------
+
+Another quick way to install CakePHP is `Oven <https://github.com/CakeDC/oven>`_. 
+It is a simple PHP script which checks the necessary system requirements,
+installs the CakePHP application skeleton and sets the development environment up.
+
+* Upload the oven.php to your server directory.
+* Execute the oven.php via your browser and click on the big oven icon.
```

Comment:
I mean we already linked to it, I dont see the need to replicate the README instructions here.
We can omit this part, and start again with "After the installation completes".